c2cgeoportal/views/entry.py

```diff
@@ -584,7 +584,9 @@ def _fill_internal_wms(self, l, layer, wms, wms_layers, errors):
         else:
             if self.default_ogc_server.type != OGCSERVER_TYPE_GEOSERVER:
                 errors.add(
-                    "The layer '{0!s}' is not defined in WMS capabilities".format(wmslayer)
+                    "The layer '{}' ({}) is not defined in WMS capabilities from '{}'".format(
+                        wmslayer, layer.name, self.default_ogc_server.name
```

Comment:
layer.layer is the same as `wmslayer` than it's displayed, and `layer.name` can be helpful when we have more than one layer with the same `layer`...

I don't understand, this is for layerv1, for layerwms and layerwmts it's working with layer(s) and name for a long time :-)